MelonLoader/Melons/Events/MelonAction.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace MelonLoader
+{
+    internal class MelonAction
+    {
+        internal readonly MethodInfo method;
+        internal readonly object obj;
+        internal readonly bool unsubscribeOnFirstInvokation;
+
+        internal MelonBase Melon => obj is MelonBase melon ? melon : null;
+
+        private MelonAction(Delegate singleDel, bool unsubscribeOnFirstInvokation)
```

Comment:
Invokation -> Invocation everywhere in this class

Reply:
love english

Reply:
https://github.com/SlidyDev/MelonLoader/commit/090153abfc48d0ad44176894c6bbc89eeed815c7